homeassistant/components/jewish_calendar/binary_sensor.py

```diff
@@ -62,7 +78,15 @@ def __init__(
     @property
     def is_on(self) -> bool:
         """Return true if sensor is on."""
-        return cast(bool, self._get_zmanim().issur_melacha_in_effect)
+        zmanim = self._get_zmanim()
+        if self.entity_description.key == "issur_melacha_in_effect":
+            return cast(bool, zmanim.issur_melacha_in_effect)
```

Comment:
We can add these as lambda's to the entity description instead, that way we don't have the make if/if/if trees here.

Reply:
Can you kindly point me to an example?